Dart/src/com/jetbrains/lang/dart/DartBundle.properties

```diff
@@ -260,11 +260,11 @@ rename.need.reRun=Cannot rename.\nThere were changes in code after usages have b
 
 exportable.DartCodeFoldingSettings.presentable.name=Code Folding Settings
 
-analysis.server.status.good.text=Send feedback
+analysis.server.status.good.text=Dart Plugin: Submit Feedback
```

Comment:
Both lgtm. This is inside of a Dart Analysis tool window, so probably it's ok to skip one more 'Dart' word. But I'll accept any.